mkdocs.yml

```
@@ -15,6 +15,10 @@ nav:
         - 'Step 추가':
           - List: 'eai/functions/add_step.md'
           - 'RFC Step 추가': 'eai/functions/add_rfc_step.md'
+    - LC5:
+      - Overview: 'lc5/index.md'
+      - Functions: 'lc5/functions/index.md'
+        
```

아래와 같은 구조로 문서를 만들어주세요.

```suggestion
      - "미리 정의된 컴포넌트 라이브러리": 'lc5/prebuilt_components.md'
      - "미리보기": 'lc5/preview.md'
      - "반응형 디자인 지원": "lc5/responsive.md"
      - "데이터 바인딩": "lc5/binding.md"
      - "템플릿 및 테마": "lc5/templates.md"
      - "버전 관리": "lc5/versioning.md"
```